example/README.md

```diff
@@ -10,20 +10,17 @@
 ### Install
 1. Clone the `stripe-android` repository.
 2. Open the project in Android Studio.
-3. After [deploying the example backend to Heroku](#deploy-the-example-backend-to-heroku) and [configuring the app](#configure-the-app), build and run the project.
+3. After [remixing the Glitch project](#remix-the-example-project-on-glitch) and [configuring the app](#configure-the-app), build and run the project.
 
 <img width="215" height="108" src="https://raw.githubusercontent.com/stripe/stripe-android/master/example/images/run.png" />
 
-### Deploy the example backend to Heroku
-1. [Create a Heroku account](https://signup.heroku.com/) if you don't have one.
-2. Navigate to the [example mobile backend repo](https://github.com/stripe/example-mobile-backend/tree/v19.0.0)
-   and click "Deploy to Heroku".
-3. Set an _App Name_ of your choice (e.g. Stripe Example Mobile Backend).
-4. Under _Config Vars_, set your [Stripe testmode secret key](https://dashboard.stripe.com/test/apikeys)
-   for the `STRIPE_TEST_SECRET_KEY` field.
-5. Click "Deploy for Free".
-
-<img width="700" height="793" src="https://raw.githubusercontent.com/stripe/stripe-android/master/example/images/heroku.png" />
+### Remix the example project on Glitch
+We provide an example backend hosted on Glitch, allowing you to easily test an integration end-to-end.
```

Comment:
```suggestion
We provide an example backend hosted on Glitch, allowing you to easily test an integration end-to-end. You can also [remix](https://glitch.happyfox.com/kb/article/23-remix/) it to make it your own.
```

Reply:
They need to remix, so I wouldn't say `You can also remix`